README.md

```diff
@@ -88,42 +138,42 @@ A common transaction type is the transfer of asset ownership from one user to an
 
 #### What are operations that require authorization?
 
-These operations add to or update data in Immutable's databases and typically require the authorization of an object's owner (ie. a user creating a project on Immutable X).
+These operations add to or update data in Immutable's databases and typically require the authorization of an object's owner (ie. a user creating a project on ImmutableX).
 
 ### How do applications generate and use signers?
 
 Signers are abstractions of user accounts that can be used to sign transactions. A user's private key is required to generate them.
 
 There are two ways to get signers in your application:
-
 1. [Generate your own by obtaining and using the user's private keys](#1-generate-your-own-signers)
 2. [Use our Wallet SDK to connect to a user's wallet application](#2-generate-signers-using-the-wallet-sdk)
 
 The first option, where an application obtains a user's private key directly, is risky because these keys allow anyone in possession of them full control of an account.
 
 The second option provides an application with an interface to the user's account by prompting the user to connect with their wallet application (ie. mobile or browser wallet). Once connected the app can begin asking the user to sign transactions and messages without having to reveal their private key.
 
-As Immutable X enables applications to execute signed transactions on both Ethereum (layer 1) and StarkEx (layer 2), signers are required for both these layers.
 
-### 1. Generate your own signers
 
-The Core SDK provides functionality for applications to generate Stark (L2) [private keys](/src/utils/stark/starkCurve.ts#L108) and [signers](/src/utils/stark/starkSigner.ts#L60).
+### 1. Generate your own signers
 
-#### 🚨🚨🚨 Warning 🚨🚨🚨
-> If you generate your own Stark private key, you will have to persist it. The key is [randomly generated](/src/utils/stark/starkCurve.ts#L108) so **_cannot_** be deterministically re-generated.
+The Core SDK provides functionality for applications to generate Stark (L2) [signers](/src/utils/stark/starkSigner.ts#L60).
 
 ```ts
 import { AlchemyProvider } from '@ethersproject/providers';
 import { Wallet } from '@ethersproject/wallet';
-import { generateStarkPrivateKey, createStarkSigner } from '@imtbl/core-sdk';
+import { generateStarkPrivateKey, generateLegacyStarkPrivateKey, createStarkSigner } from '@imtbl/core-sdk';
```

Comment:
I prefer not to import the `generateStarkPrivateKey` and the `generateLegacyStarkPrivateKey` in this example. 

We should assume the Stark key is retrieved similarly to how their ETH key is retrieved in this line `const ethSigner = new Wallet(YOUR_PRIVATE_ETH_KEY).connect(provider);`

Reply:
`YOUR_PRIVATE_ETH_KEY` is supplied in that line.